rpm/xl2tpd.spec

```
@@ -21,14 +21,15 @@ xl2tp Layer 2 Tunnelling Protocol Daemon.
 %autosetup -p1 -n %{name}-%{version}/%{name}
 
 %build
-make %{?jobs:-j%jobs}
+%make_build
```

append PREFIX="%{_usr}" here and then you can skip the patch.

---

Doesn't seem to help here. Fedora passes prefix on make install, but that doesn't seem to work for me either.

---

That's because of our %make_install macro ignores parameters. You have to call that manually.
Like this: 
```suggestion
%{__make}  DESTDIR=%{buildroot} PREFIX=%{_prefix} install
```

See:
https://github.com/sailfishos/meego-rpm-config/blob/master/macros#L81

---

Nope, doesn't work either. The patch was already there and while it's nice to get rid of them, the benefit on small repository like this is not necessarily much. I'll not waste more time on that now.